### DISCUSSION TITLE
ScriptReference/Unity.Collections.Allocator.Persistent

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#5] [Notes]
# ScriptReference/Unity.Collections.Allocator.Persistent

https://docs.unity3d.com/ScriptReference/Unity.Collections.Allocator.Persistent.html

**Comment by vertxxyz** [comment]
**`Allocator.Persistent`** is the slowest allocation but can last as long as you need it to, and if necessary, throughout the application’s lifetime. It is a wrapper for a direct call to [`malloc`](https://cplusplus.com/reference/cstdlib/malloc/). Longer jobs can use this `NativeContainer` allocation type. Don’t use `Persistent` where performance is essential.
(See [NativeContainer](https://docs.unity3d.com/Manual/JobSystemNativeContainer.html).)
